packages/landingpage/pages/index.tsx

```diff
@@ -0,0 +1,46 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+import { useEffect, useState } from "react";
+import styles from "../styles/Home.module.css";
+
+const ConvertKit: React.FunctionComponent = () => {
+  const [loaded, setLoaded] = useState(false);
+  useEffect(() => {
+    setLoaded(true);
+  }, [setLoaded]);
+  return (
+    <>
+      {!loaded && (
+        <Head>
+          <script
+            async
+            src="https://prodigious-trader-7332.ck.page/a85e477729/index.js"
```

Comment:
This currently points to the RoamJS convertkit. Once we create a form within the CabinDAO converkit, we could swap out the ids.

Also, we need to do this `loaded` hack since the convertkit embed is not really compatible with SSR, and we need to wait to run it after document hydration